content/docs/guides/branch-restore.md

```diff
@@ -0,0 +1,147 @@
+
+![branch restore backup branch](/docs/guides/branch_restore_backup_file.png)
+
+To make sure you choose the right restore point, we encourage you to use Time Travel Assist _before_ running a restore job, but the backup branch is there if you need it.
+If you do need to revert your changes, you can manually make the backup branch your primary branch using the steps described in [Branching - Point in time restore](/docs/guides/branching-pitr#change-your-primary-branch). 
+
+<Admonition type="note">
+Restoring to another branch is coming soon. See our [roadmap](/docs/introduction/roadmap). Once available, you will be able to restore to any other branch, including this restore backup, using a similar one-click operation.
+</Admonition>
+
+### Performing time travel queries
+
+Here is how to use the Time Travel Assist SQL editor:
+
+1. Select the branch you want to query against, then select a timestamp, the same as you would to [Restore a branch](#restore-a-branch-to-an-earlier-state).
+
+    This makes the selection for the Time Travel query. Notice the updated fields above the SQL editor show the **branch** and **timestamp** you just selected.
+    ![Time travel query](/docs/guides/time_travel_assist.png)
+  
+1. Check that you have the right database selected to run your query against. Use the database selector under the SQL editor to switch to a different database for querying against.
+1. Write your read-only query in the editor, then click **Query at timestamp** to run the query. You don't have to include time paramaters in the query; the query is automatically targetted to your selected timestamp.
+
+If your query is successful, you will see a table of results under the editor.
+
+Depending on your query and the selected timestamp, instead of a table of results, you might see different error messages like:
+| Error                | Explanation             |
+|----------------------|-------------------------|
+| If you query a timestamp in the future         | Console request failed with 400 Bad Request: timestamp 2024-01-10 21:59:48.639 +0000 UTC is in the future, try an older timestamp |
+| If you query a timestamp from before your project was created | Console request failed with 400 Bad Request: parent timestamp 2023-10-01 20:59:48.639 +0000 UTC is earlier than the project creation timestamp 2023-11-07 18:47:53.023515 +0000 UTC, try a more recent timestamp |
+| If you query from earlier than your history retention window | Console request failed with 400 Bad Request: timestamp 2023-12-31 21:59:48.639 +0000 UTC precedes your project's history retention window of 168h0m0s, try a more recent timestamp |
+
+Adjust your selected timestamp accordingly.
+
+## Billing
+
+There are minimal impacts to billing from these two related features.
```

Comment:
```suggestion
There are minimal impacts to billing from the branch restore and Time Travel Assist features.
```

Reply:
Additional notes:

- Should we cover the limitations? 
-- You cannot delete a backup branch without first removing the restored child branch. 
-- Once you restore a branch, you cannot reset from "main"? 
- Storage costs: Eventually, the child restore branch and the backup parent branch will diverge, and the backup parent branch will continue to hold data from an earlier point in time. These parent back up branches will start to cost you as they age beyond the history retention window. Do we need to document that as a limitation? Could you add an endpoint to a parent branch backup branch and remove all data if you need to?

Reply:
These are good points. I think we probably should add a limitations section. Will get confirmation about these before adding the section.